consensus/safety-rules/src/lib.rs

```diff
@@ -24,7 +24,10 @@ pub use crate::{
     t_safety_rules::TSafetyRules,
 };
 
-#[cfg(any(test, feature = "testing"))]
+#[cfg(any(test, feature = "fuzzing"))]
+pub mod fuzzing_utils;
+
+#[cfg(any(test, feature = "fuzzing", feature = "testing"))]
 #[path = "test_utils.rs"]
```

Comment:
is that path necessary oO?

Reply:
Good question... I'm not sure why it's there.. I'll play with it in a separate PR and if not, just remove it 😄